Tools/CISupport/ews-build/steps.py

```diff
@@ -310,12 +310,16 @@ def start(self):
     def add_patch_id_url(self):
         patch_id = self.getProperty('patch_id', '')
         if patch_id:
+            self.setProperty('change_id', patch_id, 'try build')
```

Comment:
this third parameter in setProperty is source. Let's just have 'ConfigureBuild' as the source instead of 'try build' (which would have been incorrect).